### ISSUE TITLE
getMarketplaceParticipations “isParticipating”  This return value is true for unopened sites

### ISSUE THREAD

**Issue by shuolol** [amzn/selling-partner-api-docs#254]
getMarketplaceParticipations “isParticipating”  This return value is true for unopened sites

**Comment by moltar** [comment]
Yes, the meaning of this prop is unclear. And documentation has no explanation. Please improve the documentation.

https://github.com/amzn/selling-partner-api-docs/blob/d0f158330953babe476c1ab1fc28a776064bc84d/references/sellers-api/sellers.md#L156

![screenshot-20210920T023147-FkVxtgR5](https://user-images.githubusercontent.com/491247/133940523-2dcbf31f-089a-40ab-a1bf-9dd34da6afe6.png)
